docs/start/envwin.md

```diff
@@ -7,6 +7,7 @@ No known system prerequisites are known at this time.
 ## Windows 7 to Windows 8.1, Windows Server 2008 R2 SP1 to Windows Server 2012 R2 (64-bit)
 
 [PowerShell 3.0 or higher](https://msdn.microsoft.com/en-us/powershell/scripting/setup/installing-windows-powershell)
+[.NET Framework x64 4.5 or higher](https://www.microsoft.com/en-us/download/details.aspx?id=36359)
```

Comment:
change the linked url to https://docs.microsoft.com/en-us/dotnet/framework/install/

Reply:
please add a newline to the end of PowerShell requirements. :D
![image](https://user-images.githubusercontent.com/1750815/40320342-5b4e584a-5cf9-11e8-98b8-dabfd6db3561.png)

Reply:
all updated now @TingluoHuang

Reply:
@danielstocker you forgot to change the URL. :D
https://docs.microsoft.com/en-us/dotnet/framework/install/

Reply:
oh god that's what happens when you try and do stuff quickly. should be fine now @TingluoHuang